libyul/YulString.h

```diff
@@ -162,7 +162,7 @@ class YulString
 	YulStringRepository::Handle m_handle{ 0, YulStringRepository::emptyHash() };
 };
 
-inline YulString operator "" _yulstring(char const* _string, std::size_t _size)
+inline YulString operator "" _yulname(char const* _string, std::size_t _size)
```

Comment:
If we have to do this while it's still inside `YulString`, maybe it would have been better to just rename `YulString`. As a bonus we'd also be sure we did not miss any instances of `YulString` probably a better diff between `YulString` and `YulName` in later PRs.

But maybe it does not make all that much difference given we have the follow-up PRs already lined up.